### DISCUSSION TITLE
UserId not logging in Langfuse via litellm

### DISCUSSION THREAD

**Discussion by latentloop07** [open-webui/open-webui#5627] [General]
I have hosted langfuse and litellm on my local.
And using litellm proxy in open-webui. So, I am not getting User Id of open-webui logged in langfuse.

Am I missing any configurations or it is an existing issue.

I also followed the below discussions, but it was not helpful.
https://github.com/open-webui/open-webui/discussions/1014

**Comment by bleriot14** [comment]
Have you found any solution ?
